.github/workflows/ci.yml

```
@@ -0,0 +1,34 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

You could also include a scheduled run every day to detect when the package is failing due to external changes:
```suggestion
    branches: [ master ]
  schedule:
    # Run every day at 12:42am UTC.
    - cron:  '42 12 * * *'
```